tests/pytests/test_kaguya_drivers.py

```diff
@@ -1,166 +1,118 @@
-        INVALID_PIXELS                   = (0 , 0 , 0 , 0)
-    END_OBJECT                           = IMAGE
-
-    OBJECT                               = PROCESSING_PARAMETERS
-        DARK_FILE_NAME                   = "TC1_DRK_00293_02951_S_N_b05.csv"
-        FLAT_FILE_NAME                   = "TC1_FLT_00293_04739_N_N_b05.csv"
-        EFFIC_FILE_NAME                  = "TC1_EFF_PRFLT_N_N_v01.csv"
-        NONLIN_FILE_NAME                 = "TC1_NLT_PRFLT_N_N_v01.csv"
-        RAD_CNV_COEF                     = 3.790009 <W/m**2/micron/sr>
-        L2A_DEAD_PIXEL_THRESHOLD         = 30
-        L2A_SATURATION_THRESHOLD         = 1023
-        DARK_VALID_MINIMUM               = -5
-        RADIANCE_SATURATION_THRESHOLD    = 425.971000 <W/m**2/micron/sr>
-    END_OBJECT                           = PROCESSING_PARAMETERS
-    END
-    """
-
-def test_kaguya_creation(kaguya_tclabel):
-    #with KaguyaTcPds3NaifSpiceDriver(kaguya_tclabel) as m:
-    #    d = m.to_dict()
-    #    assert isinstance(d, dict)
-
-    # Need to insert new tests here, one for each property unique to this driver
-    assert True
+def driver():
+    return KaguyaTcPds3NaifSpiceDriver("")
+
+@patch('ale.base.label_pds3.Pds3Label.instrument_id', 123)
+def test_instrument_id(driver):
+    with patch.dict(driver.label, {'SWATH_MODE_ID':'NOMINAL', 'PRODUCT_SET_ID':'TC_w_Level2B0' }) as f:
+        assert driver.instrument_id == 'LISM_123_WTN'
+
+@patch('ale.base.label_pds3.Pds3Label.instrument_id', 123)
+def test_tc_id(driver):
+    assert driver._tc_id == -12345
+
+def test_clock_stop_count(driver):
+    with patch.dict(driver.label, {'CORRECTED_SC_CLOCK_STOP_COUNT':
+        pvl._collections.Units(value=105, units='<sec>')}) as f:
+        assert driver.clock_stop_count == 105
+
+def test_clock_start_count(driver):
+    with patch.dict(driver.label, {'CORRECTED_SC_CLOCK_START_COUNT':
+        pvl._collections.Units(value=501, units='<sec>')}) as f:
+        assert driver.clock_start_count == 501
+
+@patch('ale.base.data_naif.NaifSpice.ephemeris_stop_time', 800)
+@patch('ale.base.data_naif.NaifSpice.spacecraft_id', 123)
+def test_ephemeris_stop_time(driver):
+    assert driver.ephemeris_stop_time == 0.1
+
+@patch('ale.base.data_naif.NaifSpice.ephemeris_start_time', 800)
+@patch('ale.base.data_naif.NaifSpice.spacecraft_id', 123)
+def test_ephemeris_start_time(driver):
+    assert driver.ephemeris_start_time == 0.1
+
+def test_detector_center_line(driver):
+    assert driver.detector_center_line == 0
+
+@patch('ale.base.label_pds3.Pds3Label.instrument_id', 123)
+def test_detector_center_sample(driver):
+    assert driver.detector_center_sample == 0
+
+@patch('ale.base.label_pds3.Pds3Label.instrument_id', 123)
+@patch('ale.base.label_pds3.Pds3Label.target_name', 'MOON')
+def test_sensor_orientation(driver):
+    with patch('ale.base.type_sensor.LineScanner.ephemeris_time', new_callable=PropertyMock) as mock_time:
+        mock_time.return_value = np.linspace(100,200,2)
```

Comment:
Was there a reason this patch could not be pulled-out to the front along with the others?

Reply:
I'm not sure exactly what was going on but it was giving me a hard time when I was trying to do it that way. I think it might have had something to do with setting it equal to np.linespace(100,200,2)

Reply:
Oh, interesting... Ok.